docs/language/accounts.mdx

```diff
@@ -641,8 +641,28 @@ as is the behavior when a path is added or removed from storage.
 
 <Callout type="warning">
 The order of iteration is undefined. Do not rely on any particular behaviour.
+
 Saving to or removing from storage during iteration can cause the order in which values are stored to change arbitrarily. 
-Continuing to iterate after such an operation may cause iteration to repeat some elements or skip others. 
+
+Continuing to iterate after such an operation will cause Cadence to panic and abort execution. In order to avoid such errors,
+we recommend not modifying storage during iteration. If you do, return `false` from the iteration callback to cause iteration 
+to end after the mutation like so:
```

Comment:
Breaking on logical ends makes future edits/reviews easier:

```suggestion
Continuing to iterate after such an operation will cause Cadence to panic and abort execution. 
In order to avoid such errors, we recommend not modifying storage during iteration. 
If you do, return `false` from the iteration callback to cause iteration to end after the mutation like so:
```